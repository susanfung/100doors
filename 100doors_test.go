package main

import (
	"testing"
	approvals "github.com/approvals/go-approval-tests"
)

func TestInitialState(t *testing.T) {
	approvals.VerifyArray(t, Answer(0))
}

func TestFirstPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(1))
}

func TestSecondPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(2))
}