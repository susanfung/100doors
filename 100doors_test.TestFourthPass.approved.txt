[0]=opened
[1]=closed
[2]=closed
[3]=opened
[4]=opened
[5]=opened
[6]=opened
[7]=opened
[8]=closed
[9]=closed
[10]=opened
[11]=closed
[12]=opened
[13]=closed
[14]=closed
[15]=opened
[16]=opened
[17]=opened
[18]=opened
[19]=opened
[20]=closed
[21]=closed
[22]=opened
[23]=closed
[24]=opened
[25]=closed
[26]=closed
[27]=opened
[28]=opened
[29]=opened
[30]=opened
[31]=opened
[32]=closed
[33]=closed
[34]=opened
[35]=closed
[36]=opened
[37]=closed
[38]=closed
[39]=opened
[40]=opened
[41]=opened
[42]=opened
[43]=opened
[44]=closed
[45]=closed
[46]=opened
[47]=closed
[48]=opened
[49]=closed
[50]=closed
[51]=opened
[52]=opened
[53]=opened
[54]=opened
[55]=opened
[56]=closed
[57]=closed
[58]=opened
[59]=closed
[60]=opened
[61]=closed
[62]=closed
[63]=opened
[64]=opened
[65]=opened
[66]=opened
[67]=opened
[68]=closed
[69]=closed
[70]=opened
[71]=closed
[72]=opened
[73]=closed
[74]=closed
[75]=opened
[76]=opened
[77]=opened
[78]=opened
[79]=opened
[80]=closed
[81]=closed
[82]=opened
[83]=closed
[84]=opened
[85]=closed
[86]=closed
[87]=opened
[88]=opened
[89]=opened
[90]=opened
[91]=opened
[92]=closed
[93]=closed
[94]=opened
[95]=closed
[96]=opened
[97]=closed
[98]=closed
[99]=opened